.travis.mk

```diff
@@ -405,15 +403,16 @@ test_static_build_cmake_osx_github_actions: base_deps_osx_github_actions test_st
 
 deps_freebsd:
 	sudo pkg install -y git cmake gmake icu libiconv \
-		python27 py27-yaml py27-six py27-gevent
+		python37 py37-yaml py37-gevent py37-six py37-msgpack
```

Comment:
python-msgpack *package* is not used anywhere (it is shipped with test-run using a submodule).